specs/agents/mobile/metrics.md

```diff
@@ -0,0 +1,41 @@
+## Mobile Metrics
+
+### CPU metrics
+| Name               | Type             | Units      | Description                     |
+|--------------------|------------------|------------|---------------------------------|
+| `system.cpu.usage` | Observable Guage | percentage | A percentage value of cpu usage |
+
+### Memory Metrics
+| Name                   | Type             | Units | Description                             |
+|------------------------|------------------|-------|-----------------------------------------|
+| `system.memory.usage`  | Observable Guage | bytes | The application's memory usage in bytes |
```

Comment:
```suggestion
| `system.memory.usage`  | Gauge | bytes | The application's memory usage in bytes |
```

Reply:
If it's just for the application, should this be `process.memory.usage`?

https://www.elastic.co/guide/en/apm/agent/java/current/metrics.html#metrics-system

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/process-metrics.md

Reply:
> If it's just for the application, should this be `process.memory.usage`?
> 

I think both `memory.usage` and `cpu.usage` should be reported per `application` and `system`.